src/utils/userDataManager.ts

```diff
@@ -4,30 +4,58 @@ import * as path from 'path';
 import { HistoricalHttpRequest } from '../models/httpRequest';
 import { JsonFileUtility } from './jsonFileUtility';
 
+const restClientDir = 'rest-client';
+const rootPath = path.join(os.homedir(), `.${restClientDir}`);
+
+function getCachePath(): string {
+    if (fs.existsSync(rootPath)) {
+        return rootPath;
+    }
+
+    if (process.env.XDG_CACHE_HOME !== undefined) {
+        return path.join(process.env.XDG_CACHE_HOME, restClientDir);
+    }
```

Comment:
Just a question, if an existing user wants to use the XDG directory, he needs to delete the `.rest-client` folder first. And they also need to export these two environment variables in shell. right?

Reply:
Yes, the user would need to delete it and have the env variables exported. A lot of distributions have these exported by default.